components/http_kernel/introduction.rst

```diff
@@ -339,7 +364,18 @@ of arguments that should be passed when executing that callable.
 
     b) If the argument in the controller is type-hinted with Symfony's
        :class:`Symfony\\Component\\HttpFoundation\\Request` object, then the
-       ``Request`` is passed in as the value.
+       ``Request`` is passed in as the value. If you have a custom class extending
```

Comment:
I don't know exactly how to say it but is should be something like:
"If the argument in the controller is type-hinted with Symfony's `Request` class, then the `Request`` object (or child object) is passed in as the value."